### PR TITLE
Added New Lines

### PR DESCRIPTION
**Pull request by Navzter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,3 +184,11 @@ Contributions are welcome! See the [contribution guidelines](CONTRIBUTING.md).
 [![CC0](http://i.creativecommons.org/p/zero/1.0/88x31.png)](http://creativecommons.org/publicdomain/zero/1.0/)
 
 To the extent possible under law, the author has waived all copyright and related or neighboring rights to this work.
+
+#NavzterCode
```
Comment (vvscode) [inline README.md]: why header level 1? add space

File: README.md [hunk 2]
```diff
@@ -184,3 +184,11 @@ Contributions are welcome! See the [contribution guidelines](CONTRIBUTING.md).
...
+- [catapult](https://github.com/catapult-project/catapult/labels/Good%20First%20Bug) _(label: Good First Bug)_ <br> Catapult home for performance tools.
+- [Python Babel](https://github.com/python-babel/babel/labels/difficulty%2Flow) _(label: difficulty/low)_ <br> The Python Internationalization Library.
```
Comment (vvscode) [inline README.md]: split by languages

File: README.md [hunk 3]
```diff
@@ -184,3 +184,11 @@ Contributions are welcome! See the [contribution guidelines](CONTRIBUTING.md).
...
+- [Kinto](https://github.com/Kinto/kinto/labels/easy-pick) _(label: easy-pick)_ <br> A lightweight JSON storage service with synchronisation and sharing abilities.
+- [BorgBackup](https://github.com/borgbackup/borg/labels/easy) _(label: easy)_ <br> Deduplicating backup program with compression and authenticated encryption.
+- [scrapy](https://github.com/scrapy/scrapy/labels/good%20first%20issue) _(label: good first issue)_ <br>A fast high-level web crawling & scraping framework for Python.
```
Comment (vvscode) [inline README.md]: check for duplications - it's already exists

**Comment by MunGell** [comment]
Hey, sorry this PR does not follow contributing guidelines, please refer to them before creating another one.

**Pull request closed without merging**